.github/workflows/release.yml

```diff
@@ -38,3 +38,23 @@ jobs:
           name: ${{ steps.semantic_release_info.outputs.git_tag }}
           body: ${{ steps.semantic_release_info.outputs.notes }}
           target_commitish: ${{ steps.commit_and_push.outputs.latest_commit_hash }}
+
+      - name: Go Mod Tidy
+        if: ${{ steps.semantic_release_info.outputs.git_tag }}
+        run: go mod tidy
+
+      - name: Go Build
+        if: ${{ steps.semantic_release_info.outputs.git_tag }}
+        run: CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=${{ steps.semantic_release_info.outputs.git_tag }}" -a -installsuffix cgo -o bin/2ms main.go
```

Comment:
use `env` to set environment args.
also for version use `steps.semantic_release_info.outputs.version`